crates/ruma-federation-api/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ Bug fixes:
 Improvements:
 
 - Add support for authenticated media endpoints, according to MSC3916 / Matrix 1.11
+- Make `Content-Type` and `Content-Disposition` mandatory when creating media
+  responses.
```

Comment:
Here, like in the client API, we should add "according to MSC2701 / MSC2702 / Matrix 1.12"